tests/test_bfd_global.rb

```diff
@@ -39,11 +39,31 @@ def test_create_destroy
 
     # destroy
     bg.destroy
+    assert_equal(bg.interval, bg.default_interval) if bg.interval
```

Comment:
Why the `if` checks on all of these asserts? Shouldn't these all be equal to default regardless?